pkg/param/param.go

```diff
@@ -122,8 +122,9 @@ type KeyPair struct {
 
 // Phase represents a Blueprint phase and contains the phase output
 type Phase struct {
-	Secrets map[string]v1.Secret
-	Output  map[string]interface{}
+	ConfigMaps map[string]v1.ConfigMap
```

Comment:
@tdmanv @PrasadG193 - I added this so I could reference add ConfigMap references. I think we should generalize this to support any Kubernetes resource.